api/src/services/items.test.ts

```diff
@@ -24,6 +24,10 @@ vi.mock('../env', async () => {
 vi.mock('../../src/database/index', () => ({
 	default: vi.fn(),
 	getDatabaseClient: vi.fn(),
+	getDatabaseVersion: vi.fn().mockReturnValue({
```

Comment:
The mocked value here is not actually used 😅 This is necessary for mocking this module because if it does not return a value then the various usages of `getDatabaseVersion` will fail destructuring its data even tho it may not get used.
The value gets overridden later in the tests where we mock `getDatabaseClient`'s return value.

Reply:
I see! 😄 Thanks! 👍